pkg/server/plugin/nodeattestor/azuremsi/msi.go

```diff
@@ -203,11 +200,9 @@ func (p *MSIAttestorPlugin) Attest(stream nodeattestorv1.NodeAttestor_AttestServ
 	}
 
 	var selectorValues []string
-	if tenant.client != nil {
```

Comment:
is it not longer possible to have a nil client?

Reply:
If the plugin's `Configure()` fails, is it even possible to call `Attest()` on it? With this change, a nil client will result in `Configure()` producing an error.

Reply:
you are right client is required now, so removing the if looks fine